xayn-ai/src/ranker/mod.rs

```diff
@@ -3,6 +3,7 @@ use rubert::SMBert;
 
 use crate::{
     coi::{point::UserInterests, CoiSystem},
+    embedding::utils::Embedding,
```

Comment:
depending on if/how many times we need to clone the embedding in the new ranker, we could also use the new `ArcEmbedding` instead of `Embedding`.

Reply:
Good point! I will leave it as it is for now because I'm not sure yet how often we will need to clone the embedding but I will keep it in mind.